styles.css

```diff
@@ -105,3 +108,231 @@ ol a {
   flex-direction: row;
   justify-content: center;
 }
+
+/* Projects */
+.projects {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.projects__title {
+  width: 70%;
+  text-align: center;
+  font-size: 2.5em;
+  font-weight: 700;
+  font-style: normal;
+  line-height: 52px;
+  color: rgba(23, 43, 77, 1);
+}
+
+.projects__indicator {
+  border: solid rgba(54, 179, 126, 1) 2px;
+  border-radius: 24px;
+  width: 48px;
+  height: 4px;
+  margin-top: 24px;
+}
+
+.projects-container {
+  width: 95%;
+  margin-top: 38px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.projects__card {
+  border: 1px solid rgba(208, 217, 212, 1);
+  background-color: rgba(235, 240, 238, 1);
+  border-radius: 8px;
+  width: 90%;
+  margin-top: 24px;
+}
+
+.projects__card--text {
+  background-color: rgba(255, 255, 255, 1);
+  border-radius: 8px;
+  width: 100%;
+  height: 30vh;
+  margin-top: 252px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.projects__card--title {
+  width: 70%;
+  margin-top: 16px;
+  text-align: center;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 1.5em;
+  line-height: 32px;
+  color: rgba(58, 74, 66, 1);
+}
+
+.projects__card--tags {
+  margin-top: 16px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+}
+
+.projects__card--tags ul {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.projects__card--tags ul li {
+  width: auto;
+  height: 32px;
+  padding: 8px 12px;
+  background-color: rgba(235, 240, 238, 1);
+  border-radius: 4px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.65em;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: rgba(58, 74, 66, 1);
+}
+
+.projects__card--text button {
+  margin-top: 29px;
+}
+
+/* Button's styles */
+.action-btn {
+  border: none;
+  border-radius: 4px;
+  width: 122px;
+  height: 48px;
+  background-color: rgba(54, 179, 127, 1);
+  text-align: center;
+  font-size: 1em;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: rgba(255, 255, 255, 1);
+}
```

Comment:
- [ ] Your buttons look very well styled 👍. However, I suggest that you add the styling for the active, hover and pressed states of the buttons as indicated in the figma design. I will add a screenshot below: 

![image](https://github.com/cuauh-cabrera/my_portfolio/assets/107795045/38cdab2d-9572-45d5-b82b-bce347d3fd42)